google-services/SigninQuickstart/SigninQuickstart/SigninQuickstart.csproj

```diff
@@ -139,19 +140,9 @@
     <AndroidResource Include="Resources\values-land\dimens.xml" />
     <AndroidResource Include="Resources\values-w820dp\dimens.xml" />
   </ItemGroup>
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```

Comment:
I'd like to retain this target to ensure these files exist before we build.

Reply:
Done